frontend/.env

```diff
@@ -1,2 +1,2 @@
- DOMAIN=playlistify.jp.auth0.com
-  CLIENT_ID =naVxkso0hX6mMeahUlT8KJpfH6YNU4UZ
\ No newline at end of file
+REACT_APP_DOMAIN=playlistify.jp.auth0.com
```

Comment:
You have uploaded `.env` to github, make sure it's deleted from here added to `.gitignore`